components/intercom/conversation-closed.js

```diff
@@ -0,0 +1,54 @@
+const intercom = require("https://github.com/PipedreamHQ/pipedream/components/intercom/intercom.app.js");
+
+module.exports = {
+  name: "New Closed Conversation",
+  description: "Emits an event each time a conversation is closed.",
```

Comment:
Will need to add a `key` here and on other components, as we talked about in Slack